build/Publish/PublishNupkgToTransportFeed.targets

```diff
@@ -0,0 +1,64 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), 'Common.props'))\Common.props" />
+
+  <PropertyGroup>
+    <!-- Capture 'OutputPath' before Publish.csproj modifies it -->
+    <OutputPathTransportFeed>$(OutputPath)</OutputPathTransportFeed>
+    <BuildTasksFeedDllVersion>1.0.0-prerelease-01929-02</BuildTasksFeedDllVersion>
+    <BuildTasksFeedDll>$(NuGet_Packages)\Microsoft.DotNet.Build.Tasks.Feed\$(BuildTasksFeedDllVersion)\lib\desktop\Microsoft.DotNet.Build.Tasks.Feed.dll</BuildTasksFeedDll>
+  </PropertyGroup>
+
+  <UsingTask TaskName="PushToBlobFeed" AssemblyFile="$(BuildTasksFeedDll)" />
+
+  <ItemGroup>
+    <NupkgsForPublishing Include="$(OutputPathTransportFeed)Packages\Microsoft.NET.Sdk.*.nupkg" />
+    <NupkgsForPublishing Include="$(OutputPathTransportFeed)Packages\Microsoft.NET.Build.Extensions.*.nupkg" />
+  </ItemGroup>
+
+  <PropertyGroup>
+    <RelativePath>packages</RelativePath>
+    <TransportFeedContainerName>$(TRANSPORTFEED_STORAGE_CONTAINER)</TransportFeedContainerName>
+    <TransportFeedContainerName Condition="'$(TransportFeedContainerName)' == ''">dotnet-core</TransportFeedContainerName>
+    <TransportFeedCloudDropAccessToken>$(TRANSPORTFEED_STORAGE_KEY)</TransportFeedCloudDropAccessToken>
+    <TransportFeedCloudDropAccountName>$(TRANSPORTFEED_STORAGE_ACCOUNT)</TransportFeedCloudDropAccountName>
+    <TransportFeedCloudDropAccountName Condition="'$(TransportFeedCloudDropAccountName)' == ''">dotnetfeed</TransportFeedCloudDropAccountName>
+  </PropertyGroup>
+
+  <Target Name="PublishNupkgToTransportFeed"
+          Condition=" '$(PUBLISH_NUPKG_TO_TRANSPORT_FEED)' == 'true' "
+          DependsOnTargets="RestorePackageForTransportFeed;
+                            PushNupkgToTransportFeed" />
+
+  <Target Name="RestorePackageForTransportFeed">
+    <PropertyGroup>
+      <NewLineTF>
+        <![CDATA[
+]]>
+      </NewLineTF>
+      <SetNuget_PackagesTF>set NUGET_PACKAGES=$(NuGet_Packages)</SetNuget_PackagesTF>
+      <SetDotnet_Skip_FirstTime>set DOTNET_SKIP_FIRST_TIME_EXPERIENCE=true</SetDotnet_Skip_FirstTime>
```

Comment:
Seems like these types of property should be defined in common.props.  ie NewLineTF, SetDotnet_Skip_FirstTime, etc...

Reply:
This is a stand-alone task - and as such, should be as self-contained as possible.